src/cli.py

```diff
@@ -51,8 +51,8 @@ def cli():
                 data[0] == sysVals.SERVICES.get('SCHEDULER').get('subservice').get('DELETE_SCHEDULE').get('subPort') or
                 data[0] == sysVals.SERVICES.get('SCHEDULER').get('subservice').get('REPLACE_SCHEDULE').get('subPort'))
                 ):
-
-                embeddedCSPObj = groundStation.getEmbededCSPData(data)
+                filename = opts.getOptions()
```

Comment:
This isn't quite how the options class works

Reply:
yeah this was my experiment...it should be correct now